### PR TITLE
update(tests): fix a broken socketcall test

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind failing-test

**Any specific area of the project related to this PR?**

/area tests

**Does this PR require a change in the driver versions?**

No

**What this PR does / why we need it**:

This PR fixes a missing parameter in socketcall tests (https://github.com/falcosecurity/libs/pull/1059/files) and tries to increase the stability of dynamic snaplen tests

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp [hunk 1]
```diff
@@ -193,9 +193,12 @@ TEST(SyscallExit, socketcall_connectX)
 	/* The client performs a `connect` so the client is the src. */
 	evt_test->assert_tuple_inet_param(2, PPM_AF_INET, IPV4_CLIENT, IPV4_SERVER, IPV4_PORT_CLIENT_STRING, IPV4_PORT_SERVER_STRING);
 
+	/* Parameter 3: fd (type: PT_FD) */
+	evt_test->assert_numeric_param(3, (int64_t)client_socket_fd);
```
Comment (hbrueckner) [inline test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp:197]: Awesome! I would about to open a PR for this as well.

Comment (hbrueckner) [reply test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp:197]: @Andreagit97 While at the tests, could you cherry pick [update(tests): correct snaplen HTTP checks on s390x ](https://github.com/hbrueckner/falcosecurity-libs/commit/557e56eba4a79710ffdf2f0ebcb654382667775c) to this series? It corrects an endianess issue in the modern/old BPF drivers when comparing `BPF_HTTP_*` constants against a character array.

Comment (Andreagit97) [reply test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp:197]: Sure I will add it! I was just investigating the same failing tests :laughing:

**Review by FedeDP — APPROVED** [review]
/approve

File: test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp (hunk 1 above)
Comment (hbrueckner) [reply test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp:197]: @Andreagit97 Thanks!

/approve

**Review by hbrueckner — APPROVED** [review]

**Pull request merged**